packages/taquito-beacon-wallet/src/taquito-beacon-wallet.ts

```diff
@@ -247,10 +276,10 @@ export class BeaconWallet implements WalletProvider {
       delete operatedParams.fee;
     }
     if (!params.storageLimit) {
-      delete operatedParams.storage_limit;
+      delete operatedParams.storageLimit;
```

Comment:
for example line93, createTransferOperation has converted storageLimit to storage_limit before passing to removeDefaultParams, so the original storage_limit  is good so is gas_limit